controllers/drplacementcontrol_controller_test.go

```diff
@@ -1753,6 +1774,11 @@ func verifyInitialDRPCDeployment(userPlacement client.Object, preferredCluster s
 	Expect(latestDRPC.GetAnnotations()[controllers.LastAppDeploymentCluster]).To(Equal(preferredCluster))
 	Expect(latestDRPC.GetAnnotations()[controllers.DRPCAppNamespace]).
 		To(Equal(getVRGNamespace(userPlacement.GetNamespace())))
+
+	ns := extractNamespaceFromManifestWork(latestDRPC.Name, getVRGNamespace(latestDRPC.Namespace),
+		East1ManagedCluster)
+	Expect(ns).ToNot(BeNil())
+	Expect(ns.Labels[rmnutil.OCMBackupLabelKey]).To(Equal(""))
```

Comment:
We need to check the labels in the MW and not the manifest that it deploys, correct?

Reply:
Hmm, the second time this happened :)

Reply:
done